packages/dx-react-scheduler-material-ui/src/templates/all-day-panel/cell.jsx

```diff
@@ -9,7 +9,7 @@ import { getBorder } from '../utils';
 const styles = theme => ({
   cell: {
     padding: 0,
-    height: theme.spacing(7),
+    height: `${theme.spacing(7)}px`,
```

Comment:
A customer can customize spacing using other units such as rem. Will it work?
https://material-ui.com/customization/spacing/

Reply:
@ushkal you are right. We should revert to the old code.